module/EntraBeta/AdditionalFunctions/Get-EntraContext.ps1

```diff
@@ -1,15 +1,9 @@
-# ------------------------------------------------------------------------------
-#  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
-# ------------------------------------------------------------------------------
-@{
-    SourceName = "Get-AzureADCurrentSessionInfo"
-    TargetName = $null
-    Parameters = $null
-    Outputs = $null
-    CustomScript = @'
+function Get-EntraContext{
```

Comment:
Add the MIT license. See this file
https://github.com/microsoftgraph/entra-powershell/blob/main/module/Entra/customizations/Add-EntraGroupOwner.ps1